src/applications/gi/components/content/StemScholarshipNotification.jsx

```diff
@@ -1,14 +1,16 @@
 import React from 'react';
+import environment from '../../../../platform/utilities/environment';
```

Comment:
`vets-website`'s webpack config has an alias for `platform` folder so this can be `platform/utilities/environment` if you want.

Reply:
Thanks for the tip Rian I will share with the team.